### PR TITLE
Use seperate vars for different keys (public/private/authorized)

### PR DESCRIPTION
**Pull request by mvdriel**

### CODE REVIEW

File: vars/main.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # vars file for ssh-keys
 ---
 ssh_keys_sshdir: '.ssh'
+
+ssh_keys_private_key_filename: 'blabla-id_rsa'
```
Comment (tersmitten) [inline vars/main.yml]: blabla?

File: README.md [hunk 2]
```diff
@@ -10,21 +10,24 @@ None
...
-* `ssh_keys_users.{n}.public_keys.{n}.dest`: [required]: The remote path of the key (relative to `home/.ssh/`)
-* `ssh_keys_users.{n}.public_keys.{n}.state`: [default: `present`]: State
-* `ssh_keys_users.{n}.authorized_keys`: [default: `[]`]: Authorized keys declarations
-* `ssh_keys_users.{n}.authorized_keys.{n}.src`: [required]: The local path of the key
-* `ssh_keys_users.{n}.authorized_keys.{n}.state`: [default: `present`]: State
+* `ssh_keys_private_keys`: [default: `[]`]: Private key declarations
+* `ssh_keys_private_keys.{n}.owner`: [required]: The name of the user that should own the file
+* `ssh_keys_private_keys.{n}.group`: [optional, default `owner`]: The name of the group that should own the file
+* `ssh_keys_private_keys.{n}.src`: [required]: The local path of the key
+* `ssh_keys_private_keys.{n}.dest`: [optional, default: 'id_rsa']: The remote path of the key (relative to `home/.ssh/`)
```
Comment (tersmitten) [inline README.md]: Please use backticks for the default values

**Comment by tersmitten** [comment]
I'll try to get it merged tonight.

**Comment by mvdriel** [comment]
@tersmitten Thanks!

**Pull request merged**